tests/transport/udp/_output_session.py

```diff
@@ -8,16 +8,23 @@
 import pytest
 from pytest import raises
 import pycyphal
+import logging
 from pycyphal.transport import OutputSessionSpecifier, MessageDataSpecifier, Priority
 from pycyphal.transport import PayloadMetadata, SessionStatistics, Feedback, Transfer
 from pycyphal.transport import Timestamp, ServiceDataSpecifier
 from pycyphal.transport.udp._session._output import UDPOutputSession, UDPFeedback
+from pycyphal.transport.udp._ip._endpoint_mapping import CYPHAL_PORT
+from pycyphal.transport.commons.crc import CRC32C
+
+TransferCRC = CRC32C
```

Comment:
Please import this from pycyphal.transport.commons.high_overhead_transport please